Description: Documentation fixes
 This patch gives us a complete documentation build, and also fixes
 some minor inaccuracies in the docs.
Author: Jeff Licquia <licquia@debian.org>
Last-Update: 2011-02-20

--- htmldoc-1.8.27.orig/doc/Makefile
+++ htmldoc-1.8.27/doc/Makefile
@@ -40,7 +40,7 @@ SOURCES	=	intro.html 1-install.html 2-st
 		d-compile.html
 
 DOCUMENTS =	htmldoc.html htmldoc.pdf htmldoc.ps
-DOCFILES =	help.html htmldoc.pdf
+DOCFILES =	help.html htmldoc.pdf htmldoc.html htmldoc.ps
 
 HTMLDOC	=	../htmldoc/htmldoc$(EXEEXT) --datadir .. --strict --verbose
 
--- htmldoc-1.8.27.orig/doc/htmldoc.man
+++ htmldoc-1.8.27/doc/htmldoc.man
@@ -80,9 +80,11 @@ Specifies the bottom margin in points (n
 (##in), centimeters (##cm), or millimeters (##mm).
 
 .TP 5
---charset \fI{8859-1...8859-15}\fR
+--charset \fIcharset-id\fR
 .br
-Specifies the ISO character set to use for the output.
+Specifies the ISO character set to use for the output.  Supported
+charsets include some Windows code pages (cp-###), ISO 8859 sets 
+1-9, 14, and 15 (iso8859-##), and koi8-r.
 
 .TP 5
 --color
@@ -579,7 +581,7 @@ program.
 .SH SEE ALSO
 HTMLDOC Software Users Manual
 .LP
-http://www.easysw.com/htmldoc/
+http://www.htmldoc.org/documentation.php
 .SH AUTHOR
 Michael Sweet, Easy Software Products
 .SH TRADEMARKS
